src/cargo/util/config/de.rs

```diff
@@ -195,11 +195,35 @@ impl<'de, 'config> de::Deserializer<'de> for Deserializer<'config> {
         }
     }
 
+    fn deserialize_enum<V>(
+        self,
+        name: &'static str,
+        _variants: &'static [&'static str],
+        visitor: V,
+    ) -> Result<V::Value, Self::Error>
+    where
+        V: de::Visitor<'de>,
+    {
+        if name == "Strip" {
+            let cv = self.config.get_cv(&self.key)?;
```

Comment:
I've used `get_cv` because `get_string` requires a `&str` parameter, instead of a `&config::Key`

Reply:
I think `get_string_priv` should do what is needed.